src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_NUMBER = 100;
+
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Random random = new Random();
```

Comment:
it's bad practice to initialize dependencies in the method. you can initialize it in fields (like in this example ):
<img width="631" alt="image" src="https://user-images.githubusercontent.com/96740815/192213454-585f324a-7eed-4965-abf9-cf54a55439d8.png">
 but good practice initialization using a constructor:
```
    private final Random random;
    private final ColorSupplier supplier;

    public Lottery(Random random, ColorSupplier supplier) {
        this.random = random;
        this.supplier = supplier;
    }
```